datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/corpgroup/mappers/CorpGroupEditablePropertiesMapper.java

```diff
@@ -29,6 +31,9 @@ public CorpGroupEditableProperties apply(
     result.setDescription(corpGroupEditableInfo.getDescription(GetMode.DEFAULT));
     result.setSlack(corpGroupEditableInfo.getSlack(GetMode.DEFAULT));
     result.setEmail(corpGroupEditableInfo.getEmail(GetMode.DEFAULT));
+    result.setPictureLink(
+        (Objects.requireNonNull(corpGroupEditableInfo.getPictureLink(GetMode.DEFAULT))).toString());
+
```

Comment:
what happens for existing cases where picture link might not be set? Have you tested opening the UI for an existing group?